hummingbird/ml/_parse.py

```diff
@@ -514,12 +514,12 @@ def _get_column_index(i, inputs):
                     "dimension.".format(i, inputs[vi])
                 )
             end += rel_end
+    elif isinstance(i, str):
+        for ind, inp in enumerate(inputs):
+            if inp.onnx_name == i:
+                return ind, 0
+        raise RuntimeError("Unable to find column name '{0}'".format(i))
     else:
-        # TODO: implement strings
-        # for ind, inp in enumerate(inputs):
-        #    if inp.onnx_name == i:
-        #        return ind, 0
-        # raise RuntimeError("Unable to find column name '{0}'".format(i))
```

Comment:
You can make line 517 be "else" and then delete line 523-525.  there are only two cases.  At least based on [skl2onnx](https://github.com/onnx/sklearn-onnx/blob/9939c089a467676f4ffe9f3cb91098c4841f89d8/skl2onnx/common/utils.py#L102)

Reply:
fixed.